LayoutTests/inspector/page/overrideUserPreference.html

```diff
@@ -0,0 +1,131 @@
+            let testCases = [
+                {
+                    expectedValue: "off",
+                    expectedMatchMedia: false,
+                },
+                {
+                    preferenceName: InspectorBackend.Enum.Page.UserPreferenceName.PrefersReducedMotion,
+                    preferenceValue: InspectorBackend.Enum.Page.UserPreferenceValue.On,
+                    expectedValue: "on",
+                    expectedMatchMedia: true,
+                },
+                {
+                    preferenceName: InspectorBackend.Enum.Page.UserPreferenceName.PrefersReducedMotion,
+                    preferenceValue: InspectorBackend.Enum.Page.UserPreferenceValue.Off,
+                    expectedValue: "off",
+                    expectedMatchMedia: false,
+                },
+                {
+                    preferenceName: InspectorBackend.Enum.Page.UserPreferenceName.PrefersReducedMotion,
+                    preferenceValue: null,
+                    expectedValue: "off",
+                    expectedMatchMedia: false,
+                },
+            ];
+
+            await testOverridePreference({cssPropertyName, mediaQuery, testCases});
+        },
+    });
+
+    suite.addTestCase({
+        name: "Page.overrideUserPreference.PrefersContrast",
+        description: "",
+        async test() {
+            let cssPropertyName = "--contrast";
+            let mediaQuery = "(prefers-contrast)";
+            let testCases = [
+                {
+                    expectedValue: "off",
+                    expectedMatchMedia: false,
+                },
+                {
+                    preferenceName: InspectorBackend.Enum.Page.UserPreferenceName.PrefersContrast,
+                    preferenceValue: InspectorBackend.Enum.Page.UserPreferenceValue.On,
+                    expectedValue: "on",
+                    expectedMatchMedia: true,
+                },
+                {
+                    preferenceName: InspectorBackend.Enum.Page.UserPreferenceName.PrefersContrast,
+                    preferenceValue: InspectorBackend.Enum.Page.UserPreferenceValue.Off,
+                    expectedValue: "off",
+                    expectedMatchMedia: false,
+                },
+                {
+                    preferenceName: InspectorBackend.Enum.Page.UserPreferenceName.PrefersContrast,
+                    preferenceValue: null,
+                    expectedValue: "off",
+                    expectedMatchMedia: false,
+                },
+            ];
+
+            await testOverridePreference({cssPropertyName, mediaQuery, testCases});
+        },
+    });
+
+    suite.runTestCasesAndFinish();
+}
+
+</script>
+</head>
+<body onload="runTest()">
+    <p>Tests for the Page.overrideUserPreference command.</p>
+
+    <style>
+    body {
+        --motion: off;
+        --contrast: off;
+    }
+
+    @media (prefers-reduced-motion) {
+        body {
+            --motion: on;
```

Comment:
Nit: strictly speaking this would be turning motion off. Maybe `--test-prefers-reduced-motion: on;`? (Ditto for `--test-prefers-contrast` as well for consistency.)

Reply:
Indeed. This makes it clearer.